docs/deploy/local.md

```diff
@@ -66,14 +64,11 @@ helm upgrade --install -f ./chart/values.yaml -f ./chart/examples/local-config.y
 
 The local setup includes the full Browsertrix Cloud system, with frontend, backend api, db (via MongoDB) and storage (via Minio)
 
-An admin user with name `admin@example.com` and password `PASSW0RD!` will be automatically created.
```

Comment:
This should be true! And is working for me, without needing to uncomment anything locally.